assume/markets/clearing_algorithms/complex_clearing.py

```diff
@@ -217,11 +217,16 @@ def primal_eql_dual(m):
     eps = 1e-4
     solvers = check_available_solvers(*SOLVERS)
 
-    solver = SolverFactory(solvers[0])
+    solver = (
+        SolverFactory("gurobi") if "gurobi" in solvers else SolverFactory(solvers[0])
+    )
     if solver.name == "gurobi":
         options = {"cutoff": -1.0, "eps": eps}
     else:
         options = {}
+        raise ValueError(
+            f"Solver {solver} does not support cutoff, might not find solution."
```

Comment:
in other cases, setting a timelimit might be good, which returns the best solution found

Reply:
I'm not that familiar with all the solver options, but yea, it should be possible to run it with different solvers. Maybe we could also implement the option to choose solver settings in the configs? At least the time limit could be a nice parameter to enter manually.